app/views/catalog/ds.html.erb

```diff
@@ -26,7 +26,7 @@
                @obj.datastreams[params[:dsid]].respond_to?(:ng_xml) ||
                params[:dsid] == 'RELS-EXT'
              ) &&
```

Comment:
Can you drop this conditional too, all of the datastreams you specified match this condition.